PYME/recipes/surface_fitting.py

```diff
@@ -300,3 +304,68 @@ def execute(self, namespace):
 
         namespace[self.output_name] = shell
         namespace[self.output_name_mapped] = points
+
+
+@register_module('ImageMaskFromSphericalHarmonicShell')
+class ImageMaskFromSphericalHarmonicShell(ModuleBase):
+    """
+
+    Parameters
+    ----------
+    input_shell: spherical_harmonics.ScaledShell()
+        input localizations to fit a shell to
+    bounds_source: PYME.IO.tabular
+        optional input to estimate image bounds from, otherwise the points
+        used to fit the shell are used
+    voxelsize_nm: list
+        x, y, z pixel size in nm
+
+
+    Returns
+    ------
+    output: PYME.IO.image.ImageStack
+        boolean mask True inside, False outside
+    """
+    input_shell = Input('harmonic_shell')
+    input_image_bound_source = Input('input')
```

Comment:
Should this be more generic - e.g. able to match an existing image rather than just points ...